src/vs/workbench/browser/parts/editor2/nextEditorGroupView.ts

```diff
@@ -111,9 +113,11 @@ export class NextEditorGroupView extends Themable implements INextEditorGroupVie
 		@IPartService private partService: IPartService,
 		@INotificationService private notificationService: INotificationService,
 		@ITelemetryService private telemetryService: ITelemetryService,
-		@IUntitledEditorService private untitledEditorService: IUntitledEditorService
+		@IUntitledEditorService private untitledEditorService: IUntitledEditorService,
+		@IConfigurationService private configurationService: IConfigurationService,
 	) {
 		super(themeService);
+		this.workbenchEditorOptions = getEditorPartOptions(this.configurationService.getValue<IWorkbenchEditorConfiguration>());
```

Comment:
I suggest to just use the accessor here to get to the editor options instead of keeping it as variable.